docs/operations-guide/migrating-from-h2.md

```diff
@@ -15,15 +22,22 @@ export MB_DB_PORT=5432
 export MB_DB_USER=<username>
 export MB_DB_PASS=<password>
 export MB_DB_HOST=localhost
-java -jar metabase.jar load-from-h2 /path/to/metabase.db # do not include .mv.db or .h2.db suffix
+java -jar metabase.jar load-from-h2 /path/to/metabase.db # do not include .mv.db
 ```
 
 It is expected that you will run the command against a brand-new (empty!) database; Metabase will handle all of the work of creating the database schema and migrating the data for you.
```

Comment:
```suggestion
Metabase expects that you will run the command against a brand-new (empty) database; it will create the database schema and migrate the data for you.
```